src/cursor.rs

```diff
@@ -447,14 +274,90 @@ impl Command for BlinkOff {
     }
 }
 
-impl_display!(for Goto);
-impl_display!(for Up);
-impl_display!(for Down);
-impl_display!(for Left);
-impl_display!(for Right);
-impl_display!(for SavePos);
-impl_display!(for ResetPos);
+impl_display!(for MoveTo);
+impl_display!(for MoveUp);
+impl_display!(for MoveDown);
+impl_display!(for MoveLeft);
+impl_display!(for MoveRight);
+impl_display!(for SavePosition);
+impl_display!(for RestorePosition);
 impl_display!(for Hide);
 impl_display!(for Show);
-impl_display!(for BlinkOn);
-impl_display!(for BlinkOff);
+impl_display!(for EnableBlinking);
+impl_display!(for DisableBlinking);
+
+#[cfg(test)]
+mod tests {
+    use super::{
```

Comment:
Wrong semantic ordering ...

```diff
@@ -288,11 +287,13 @@ impl_display!(for DisableBlinking);
 
 #[cfg(test)]
 mod tests {
+    use std::io::{self, stdout, Write};
+
+    use crate::execute;
+
     use super::{
         position, MoveDown, MoveLeft, MoveRight, MoveTo, MoveUp, RestorePosition, SavePosition,
     };
-    use crate::execute;
-    use std::io::{self, stdout, Write};
 
     // Test is disabled, because it's failing on Travis
     #[test]
```